### PR TITLE
Fix 404 pages detection and support

### PR DESCRIPTION
**Pull request by bfred-it**
The DOM and page title were changed since https://github.com/sindresorhus/refined-github/pull/1558

### CODE REVIEW

File: source/libs/page-detect.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import select from 'select-dom';
 import {check as isReserved} from 'github-reserved-names';
 import {getUsername, getCleanPathname, getRepoPath, getOwnerAndRepo} from './utils';
 
-export const is404 = (): boolean => document.title === 'Page not found · GitHub';
+export const is404 = (): boolean => document.title === 'GitHub · Where software is built';
```
Comment (bfred-it) [inline source/libs/page-detect.ts]: 🤷‍♂️

Comment (vinkla) [reply source/libs/page-detect.ts]: Maybe we could use the alt text on the image? It contains `404`.

```
<img alt="404 “This is not the web page you are looking for”" class="js-plaxify position-absolute" data-xrange="20" data-yrange="10" height="249" width="271" style="z-index: 10; top: 72px; left: 72px; transform: translate3d(-6.09722px, -4.11647px, 0px);" src="data:image/png;base64,iVBOR...">
```

Comment (vinkla) [reply source/libs/page-detect.ts]: I've submitted a pull request: #1849

**Comment by bfred-it** [comment]
@lukehefson is it possible to restore the old title on 404 pages?

**Comment by lukehefson** [comment]
> lukehefson is it possible to restore the old title on 404 pages?

@bfred-it I wasn't involved in this change myself and have very little context on it– but I'll raise your comments with the relevant team now!

**Comment by bfred-it** [comment]
Thank you Luke!

**Comment by lukehefson** [comment]
Should be fixed within the next day or two 🙌

**Comment by cheshire137** [comment]
👋 The 404 page title is restored to what it was before, thanks for catching it!

**Comment by bfred-it** [comment]
Thanks lovely peeps!

Here's a thank-you avocado 🥑

**Pull request merged**